alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/model/action/Throttle.kt

```diff
@@ -65,6 +66,7 @@ data class Throttle(
                             currentToken.isValue -> {
                                 value = xcp.intValue()
                                 require(value > 0, { "Can only set positive throttle period" })
+                                require(value < MAX_THROTTLE_VALUE, { "Can only set throttle period less than 1 day" })
```

Comment:
I see on kibana max is 1440, so if user pass on 1440 it will fail here. Can we make it <= ?

Reply:
Good point, will change it.